src/scss/components/_login-options.scss

```
@@ -16,3 +16,8 @@
     @include margin(auto);
   }
 }
+
+.#{prefix(login-options__cosign)} {
+  padding: 0.6em;
+  background-color: var(--of-color-read-only-bg);
```

design tokens should always be made for the component itself, you are not allowed to use design tokens of other components (or utility tokens) for a specific goal.

So the correct approach here would be:

```suggestion
  background-color: var(--of-login-options-cosign-background-color);
```

Then, you can define that design token in https://github.com/open-formulieren/design-tokens, get a new version of that package published and use it in the SDK.